docs/content/v1/getting-started.md

```diff
@@ -3,9 +3,9 @@ title: Getting Started
 weight: 2
 ---
 
-Lego requires [npm](https://npmjs.com) or [yarn](https://yarnpkg.com) and [node](https://nodejs.org/).
+LEGO requires [npm](https://npmjs.com) or [yarn](https://yarnpkg.com) and [node](https://nodejs.org/).

```

Comment:
Should we keep `yarn` in the documentation? As now, there are a lot of other package managers that exist (`pnpm`, `bun`, ...).

As is, it is not exhaustive and adds some noise (with `(or yarn ...)` statements), while most of the time, developers using alternative package managers know it and know how to use it instead of `npm`.

Also, because `npm` is shipped by default with `node`, I don't know if it should be mentioned as a requirement in addition to `node`.

Reply:
@vinyll Any opinion on this?

Reply:
Done :ok_hand: